src/pages/Dashboard.jsx

```diff
@@ -9,27 +9,27 @@ const Dashboard = () => {
   return (
       <>
       <Header>
-            <img src={logo}/>
+            <img src={logo} alt =""/>
             Messenger
         </Header>
       <Container>
             <ChatMenu>
                     <ChatMenuWrapper>
                         <ChatMenuInput placeholder="Search contact"/>
                         <Conversation>
-                            <img src={spike}/>
+                            <img src={spike} alt=""/>
```

Comment:
Each meaningful image in the web app should have alt text to make it comfortable for ones with disabilities.

Please add alter text to user icons.
```suggestion
                            <img src={spike} alt="User avatar"/>
```